packages/compass-e2e-tests/tests/logging.test.js

```diff
@@ -9,26 +9,35 @@ function cleanLog(log) {
       '$1'
     )
   );
-  for (const entry of log) {
+  for (let i = 0; i < log.length; i++) {
+    const entry = log[i];
     expect(entry.t.$date).to.be.a('string');
     delete entry.t; // Timestamps vary between each execution
 
-    if (entry.id === 1001000001) {
+    if (entry.id === 1_001_000_001) {
       expect(entry.attr.version).to.be.a('string');
       expect(entry.attr.platform).to.equal(process.platform);
       expect(entry.attr.arch).to.equal(process.arch);
       delete entry.attr;
     }
-    if (entry.id === 1001000002) {
+    if (entry.id === 1_001_000_002) {
       entry.attr.stack = entry.attr.stack
         .replace(/file:\/\/\/.+:\d+:\d+/g, '<filename>')
         .split('\n')
         .slice(0, 2)
         .join('\n');
     }
-    if (entry.id === 1001000022 || entry.id === 1001000023) {
-      expect(entry.attr.duration).to.be.a('number');
-      entry.attr.duration = 100;
+    // Remove server heartbeat logs as they can happen a varying amount of
+    // times depending on how long tests are taking.
+    if (entry.id === 1_001_000_022 || entry.id === 1_001_000_023) {
+      log.splice(i--, 1);
```

Comment:
How do you feel about adding a `continue` here just to be intentional about moving to the next entry? Might prevent future bugs too if more cases to skip entries are added which could be non-exclusive to this or the case below.

Reply:
Will include that in the next PR :)